assets/scss/default.scss

```diff
@@ -0,0 +1,70 @@
+@import "bootstrap";
+
+.row {
+	margin: 20px auto;
+}
+
+figure img, .posters img {
+	max-width: 100%;
+}
+
+hr {
+	border-top: 1px solid #fff;
+}
+
+header {
+	&, a {color: $gray-light;}
+	h1, h2, p {
+		text-transform: uppercase;
+		letter-spacing: 1px;
+	}
+}
+
+h1, h2 {
+	font-size: $font-size-base;
+	padding: 0;
+	margin: 0;
+	line-height: 20px;
+}
+
+.poster {
+    margin: 15px 0 15px 0;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered line-height, margin<br>Shorthand form for property `margin` should be written more concisely as `15px 0` instead of `15px 0 15px 0`